### PR TITLE
Motech 2644: Updated documentation to reflect MOTECH supports IE11+

### PR DESCRIPTION
**Pull request by afijal**
Updated documentation in MOTECH to reflect that we don't support IE 9 or IE 10 anymore.

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerDatabaseServiceImpl.java [hunk 1]
```diff
@@ -261,7 +261,7 @@ private String buildActivityFilter(JobsSearchSettings jobsSearchSettings) {
         String[] activityElements = jobsSearchSettings.getActivity().split(",");
```
Comment (kmadej) [inline modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerDatabaseServiceImpl.java]: remove this changes from PR

**Comment by pgesek** [comment]
test this please

File: docs/source/development/browser.rst [hunk 2]
```diff
@@ -34,6 +34,8 @@ AngularJS 1.3 has dropped support for IE8. Read more about it on this `blog <htt
 
 The AngularJS project currently supports and will attempt to fix bugs for IE9 and above. Their continuous integration server runs all the tests against IE9, IE10, and IE11. See `Travis CI <https://travis-ci.org/angular/angular.js>`_ and `ci.angularjs.org <https://ci.angularjs.org>`_.
 
+However, MOTECH supports IE11 and above.
```
Comment (martokarski) [inline docs/source/development/browser.rst]: Could we provide better explanation why we support IE11?

File: docs/source/development/browser.rst [hunk 3]
```diff
@@ -34,6 +34,8 @@ AngularJS 1.3 has dropped support for IE8. Read more about it on this `blog <htt
...
+
 Internet Explorer 9: Issues
```
Comment (martokarski) [inline docs/source/development/browser.rst]: We do not support IE9 anymore, so I think this paragraph is redundant.

File: docs/source/development/browser.rst (hunk 2 above)
Comment (jredlarski) [inline docs/source/development/browser.rst]: @martokarski @afijal Let's add an explanation based on **Supported browsers** paragraph from this already merged [PR on the Motech UI](https://github.com/motech/motech-ui/pull/32/files) to keep both documentations as similar as possible

**Comment by jredlarski** [comment]
@martokarski @afijal Let's add an explanation why we support IE11 based on **Supported browsers** paragraph from this already merged [PR on the Motech UI](https://github.com/motech/motech-ui/pull/32/files) to keep both documentations as similar as possible

**Comment by jredlarski** [comment]
@afijal Please change the **Supported browsers** to be the same as in [PR on the Motech UI](https://github.com/motech/motech-ui/pull/32/files). Except the last line about Chromium - this should be removed. Also remove the section about the **Internet Explorer Compatibility**

**Comment by jredlarski** [comment]
test this please

**Comment by jredlarski** [comment]
@kmadej Looks good to me

**Pull request merged**